### DISCUSSION TITLE
No configuration works, causes YouTube, Spotify, any Audio App to fail.

### DISCUSSION THREAD

**Discussion by Malumen** [ExistentialAudio/BlackHole#603] [Q&A]
Most bizarre and also most useless issue report right here: _straight up kills my machine._

- Catalina 10.15.7
- Blackhole 16ch
- TASCAM Series 208i
- TASCAM US-2x2
- Built-in Output (headphones and speakers)
- DisplayPort (monitor speakers)

Every single YouTube tutorial follows the same principles outlined by @devinroth but when I select the audio output device to be the `Multi-Out Device` then my entire system cannot play audio at all. If I load a YouTube video it spins forever and recommends I restart my device. If I try to play iTunes or Spotify or anything, it will act like the song is playing for a few seconds (seconds increase on the play timer) and then eventually stops with a kind of failure error text (Spotify: "We can't play this song right now").

Even if I only select `Blackhole` and the `Built-in Output` it still gives me this crazy problem.

**Comment by devinroth** [comment]
Maybe include some screenshots to be less useless.

**Comment by Malumen** [comment]
> Maybe include some screenshots to be less useless.

Sorry Devin, a screenshot of what would help? Just a YouTube video failing to load? Or is there a code or something you're looking for?

Or one better: straight up _any_ `Multi-Output Device` cannot play. If I make a `Multi-Output Device` with just `Built-in Output` my system fails in this way too.

Strange. Maybe it is unrelated to Blackhole? I'll try uninstalling it.

<img width="1920" alt="Screen Shot 2021-06-07 at 9 57 07 PM-min" src="https://user-images.githubusercontent.com/56620641/121129853-68387980-c7e2-11eb-9ff8-58723a49594a.png">

**Comment by devinroth** [comment]
I wanted to see the multi-output.

Install BlackHole 2ch. See if that makes any difference.

**Comment by Malumen** [comment]
> I wanted to see the multi-output. Install BlackHole 2ch. See if that makes any difference.

Firstly, I tried the code you provided to remove `Blackhole-16ch` and strangely it didn't work? Gave me some outputs but after resetting coreaudiod it was all still there... and then I saw your packaged uninstaller!! DUH! Dunno why I did it the hard way...

`Blackhole-2ch` does the same thing... But weirder more: if I do _any_ sort of `Multi-Output Device` then it breaks, _not just Blackhole_. Currently nothing blackhole related is on my system and it fails in the same way when attempting any said `Multi-Output Device`...

Guess my system is messed up?
<img width="626" alt="Screen Shot 2021-06-07 at 10 52 12 PM-min" src="https://user-images.githubusercontent.com/56620641/121130641-8fdc1180-c7e3-11eb-8887-7123996b8e0e.png">

**Comment by devinroth** [comment]
Multi-outputs have always been buggy.  So who knows.

**Reply by mcjugl** [reply]
Hello! I´m Tom - and I´ve the same problems like Melumen.
My System: MacBookPro late 2013, BigSure - BlackHole16Ch
I tried to create a Multi-Output with 1.my usb-mixer and 2.iPodsPro 2
That works...but, if I want to start for example youtube - thats impossible, because the video spins for ever.
If I create only one aggregate, there is no problem - but only one output.
Whats the problem, what can I do?

Thanks a lot, Tom.

**Comment by devinroth** [comment]
There’s a whole discussion on the topic here.
https://github.com/ExistentialAudio/BlackHole/issues/758

Devin

**Comment by mcjugl** [comment]
Hello, Devin!

Thanks, Devin, for this fast information!
I have read the different comments, also your recommendation to dipper.
I understood correctly?
Dipper can capture not only different sound sources, Dipper can also
used for different outputs…in my case: AppleiPodsPro2  and my mixer-driver?
So I don’t need BlackHole?

Thanks a lot!

Tom!

> Am 27.01.2024 um 19:38 schrieb Devin Roth ***@***.***>:
> 
> 
> There’s a whole discussion on the topic here. 
> https://github.com/ExistentialAudio/BlackHole/issues/758 
> 
> Devin

**Comment by devinroth** [comment]
Yeah. macOS Bug.

Hopefully Dipper is helpful.

Devin

**Reply by mcjugl** [reply]
Hello, Devin! I tried to install Dipper from AppStore...but thats impossible, because I have BigSure...Dipper works only with MacOS 13.

Tom

**Comment by devinroth** [comment]
Shoot sorry Tom. Unfortunately there is no way for Dipper to work on earlier macOS versions.

I don’t know if there’s another solution.

Are you making sure that a 2ch audio device is set as the first and primary device in your multi-output?

Devin

**Reply by mcjugl** [reply]
That´s it! My configuration contains a mixer-driver (22ch-mixer) and the apple earPods 2 (2ch).
The 22ch-mixer was set to the first position. After changing that, it works fine.
Thanks for this important info!
One more question:
The drift-function reduces also latency?
But that will not work with bluetooth, or?

Thanks a lot, Tom

**Comment by devinroth** [comment]
I was assuming you’ve already read through all the wiki and FAQ.

Devin

**Comment by devinroth** [comment]
Drift can cause additional latency or worse audio glitches depending on which direction the drift happens.

But it won’t make your bluetooth have less latency. It will just potentially prevent more from happening.

Devin

**Reply by mcjugl** [reply]
Thanks a lot for your fast and friendly support and of course - for this helpful app! Tom
